src/components/Widgets/NumberControl.js

```diff
@@ -1,16 +1,16 @@
 import React, { PropTypes } from 'react';
 
-export default class StringControl extends React.Component {
```

Comment:
Never noticed this before, weird. I am a bit concerned about breaking current implementations which may be depending on this field producing a string.